### PR TITLE
New concepts part 2

### PR DESCRIPTION
**Pull request by CarolinHu**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
Second half of the new concepts proposed in #850 + some grapefruit. Again, if you find more NAs that fit one of the new concepts, let me know! @AnnikaTjuka, may I ask you to be the moderator?

Some issues:
- Can Monnier-2014-1031-441 be mapped to RASPBERRY? It says (mûre) in brackets, so perhaps no distinction is made between raspberry and blackberry.
- Can DiezAlamo-2018-750-418 be mapped to GRAPEFRUIT? Perhaps someone with Spanish knowledge can weigh in.
- There are some NAs with a specification of fern, should they still be mapped to the general FERN concept? That would probably mean a double mapping in Bowern.

Bodt-2019-664-176 | fern, fiddlehead [english]
Bodt-2019-664-602 | fern (not edible) [english]
Bowern-2012-644-171 | fern (tree-fern) [english]
Bowern-2012-644-172 | fern [bracken-fern] [english]
Wang-2004-471-469 | 蕨菜 [chinese]; eagle fern [english]

### CODE REVIEW

**Comment by LinguList** [comment]
Thanks, @CarolinHu. As @AnnikaTjuka is a bit busy with seminars this week, as far as I understand, I propose to ask @Schweikhard to moderate this PR this time. I recommend to ask @MacyL to join as reviewer for the Chinese concepts and volunteer to review myself.

**Comment by CarolinHu** [comment]
Ah, thanks for clearing that up and for taking a look. @Schweikhard, are you available for moderation instead?

**Comment by AnnikaTjuka** [comment]
> As @AnnikaTjuka is a bit busy with seminars this week, as far as I understand, I propose to ask @Schweikhard to moderate this PR this time.

Thanks for pointing this out @LinguList ! The first part of my workshop ended yesterday evening and next week is going to be a bit busier with two workshops in four days. But I have time to moderate this PR (it seems pretty straightforward).

Concerning Monnier-2014-1031-441, this can be mapped to RASPBERRY.

**Comment by LinguList** [comment]
Ah, one thing is important: @MacyL, these changes affect the list by Marrison in lexibank, which you added. So you should then re-run the lexibank on Marrison, once this issue is done, or make an issue that this should be done once a particular version of concepticon has appeared.

**Comment by AnnikaTjuka** [comment]
> Can DiezAlamo-2018-750-418 be mapped to GRAPEFRUIT? Perhaps someone with Spanish knowledge can weigh in.

This is an interesting case: POMELO (Spanish) 
I know pomelo as a separate fruit, see [here](https://en.wikipedia.org/wiki/Pomelo). But it is translated from Spanish to German as "Grapefruit". @LinguList Should we map this to GRAPEFRUIT or leave it unmapped?

**Comment by LinguList** [comment]
Pomelo is grapefruit for me, even in German, the definition is not straightforward. What people normally do is giving the same name to graefruit, pomelo and pampelmuse, see wiki:

> Verschiedene Zitrusfrüchte sind durch Kreuzung aus der Pampelmuse entstanden. Durch Kreuzung der Pampelmuse mit der Mandarine entstand die Orange. Die Rückkreuzung von Orange und Pampelmuse ist die Grapefruit.[2] Umgangssprachlich wird im Deutschen oft nicht zwischen Pampelmuse und Grapefruit unterschieden [wiki/Pampelmuse](https://de.wikipedia.org)/wiki/Pampelmuse

**Comment by LinguList** [comment]
What we might need is a hierarchy here: Pomelo is Pampelmuse in German (although almost nobody realizes the difference when buying them in the supermarket, I am convinced). There is also confusion in Spanish, as [wiki](https://es.wikipedia.org/wiki/Citrus_maxima) tells us again:

> Por la influencia del inglés, en ocasiones se confunde con la planta híbrida Citrus × paradisi más conocida como pomelo.

I would say, as we can be quite sure that the confusion is the normal, that we could slightly modify the definition, using GRAPEFRUIT OR POMELO as gloss. The term is very recent (citing again wiki in german):

> Man nimmt an, dass die Grapefruit um 1750 aus einer spontanen Kreuzung zwischen der Pampelmuse und der Orange auf Barbados entstanden ist. 1823 gelangte sie nach Florida, wo sie seit 1875 kommerziell angebaut wird.[7] Sie wird heute in fast allen subtropischen Ländern der Welt angebaut. Hauptproduzent sind die USA mit Plantagen in Florida und Texas. Die wichtigsten Lieferanten für Europa sind Israel, Zypern und Spanien. 

This means: we are dealing with a recent word, a new concept, and people confound it. As a result, I assume that the "concept" is similar to the idea of a crododile being the same as an alligator. We do not need to add subconcepts, since we know that in th elists where it is mapped, the speakers confuse the terms: German, and in French, the name is also pamplemousse, so the confusion is the same here.

**Comment by LinguList** [comment]
so the definition should say something like: either the fruit of the citrus whatever tree, but often also referring to the fruit of the pomelo tree, which are often not distinguished by people, due to the recent origin of the citrus whatever.

**Comment by CarolinHu** [comment]
Thanks @AnnikaTjuka for moderating, hope your workshop is going nicely.
Okay, so I will map Monnier and DiezAlamo (and also Clark-2008-441-181 pomelo [english]) and change the concept to GRAPEFRUIT OR POMELO, including the description.
Then I just need your verdict on the ferns :)

**Comment by Schweikhard** [comment]
Sorry for the late reply, yesterday was busy. Thanks for filling in for me @AnnikaTjuka!

Took a look at the changes as well and looks all good.

**Comment by AnnikaTjuka** [comment]
> There are some NAs with a specification of fern, should they still be mapped to the general FERN concept? That would probably mean a double mapping in Bowern.
Bodt-2019-664-176 | fern, fiddlehead [english]
Bodt-2019-664-602 | fern (not edible) [english]
Bowern-2012-644-171 | fern (tree-fern) [english]
Bowern-2012-644-172 | fern [bracken-fern] [english]
Wang-2004-471-469 | 蕨菜 [chinese]; eagle fern [english]

I think we usually don't map specific terms to the broad concept. But I'd like to ask @LinguList and @MacyL to comment on this as well.

**Comment by LinguList** [comment]
I don't know. For me, FERN is a rather fuzzy concept, but we cannot tell
for other areas, and we don't even know if its the same species, etc.,
so I'd say: leave them unmapped.

**Comment by MacyL** [comment]
I had a look at the definition of FERN, I was thinking if it's possible to further defined two sub concepts? 
FERN (EDIBLE) and FERN (NOT EDIBLE)?
FERN : Bowern-2012-644-171
FERN (EDIBLE) :  Bodt-2019-664-176, Wang-2004-471-469
FERN (NOT EDIBLE) : Bodt-2019-664-602, Bowern-2012-644-172

> > There are some NAs with a specification of fern, should they still be mapped to the general FERN concept? That would probably mean a double mapping in Bowern.
> > Bodt-2019-664-176 | fern, fiddlehead [english]
> > Bodt-2019-664-602 | fern (not edible) [english]
> > Bowern-2012-644-171 | fern (tree-fern) [english]
> > Bowern-2012-644-172 | fern [bracken-fern] [english]
> > Wang-2004-471-469 | 蕨菜 [chinese]; eagle fern [english]
> 
> I think we usually don't map specific terms to the broad concept. But I'd like to ask @LinguList and @MacyL to comment on this as well.

**Comment by LinguList** [comment]
we can do that, but it is also a lot of efforts for two lists that have
this? I leave it to you all to decide.

**Comment by MacyL** [comment]
true, fine by me. Then I think for the FERN case, there will be only one hit : 

FERN : Bowern-2012-644-171

> Thanks for your suggestion, @MacyL ! If we would make this distinction, it would require us to know which fern (or whether parts of it) are edible. For example, parts of the bracken fern seem to be edible as well. As a non-biologist, I'd find this difficult. Thus, I'd stick to mapping only general glosses ('fern') to FERN and leave the others unmapped.

**Comment by CarolinHu** [comment]
Thank you all for weighing in! As advised, I left the fern as it is for now and think I addressed all the other things.
Perhaps @AnnikaTjuka can have one last look at the new GRAPEFRUIT OR POMELO concept to check whether the description is not too "clumsy" :)

File: concepticondata/concepticon.tsv [hunk 1]
```diff
@@ -3783,3 +3783,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
...
+3800	IVORY	The physical world	The hard, white substance which forms the tusks of animals like elephants, narwhals, or walruses and is often considered a valuable material.	Person/Thing	
+3801	DATE (FRUIT)	Agriculture and vegetation	The sweet, edible fruit of the date palm Phoenix dactylifera of the family Arecaceae.	Person/Thing	
+3802	FERN	Agriculture and vegetation	Any member of a large group of vascular plants in the family Filices who have neither flowers nor seeds.	Person/Thing	
+3803	RASPBERRY	Agriculture and vegetation	The edible, berry-like fruit of a multitude of plant species in the genus Rubus of the family Rosaceae, most typically red, small and sweet.	Person/Thing	
+3804	GRAPEFRUIT OR POMELO	Agriculture and vegetation	The fruit of the pomelo tree, Citrus maxima, of the family Rutaceae, or the fruit of the grapefruit tree, Citrus x paradisi, a citrus hybrid which originated from crossing pomelos and oranges. Due to the recent origin of the grapefruit and the close genetic relatedness between both fruits, they are often not distinguished.	Person/Thing
```
Comment (AnnikaTjuka) [inline concepticondata/concepticon.tsv:3790]: I would change the end of the last sentence:
", they are often not differentiated lexically."
Just to make it clear that they are given the same name.

**Comment by AnnikaTjuka** [comment]
Perfect! Now everything looks good :) 
Feel free to merge @CarolinHu .

**Pull request merged**